src/form/Form.tsx

```diff
@@ -472,6 +479,20 @@ function Form({
     Object.assign(fieldValues, newFieldValues);
     // Always rerender from empty state for display purposes
     if (rerender || empty) setRender((render) => !render);
+
+    if (autoValidate) {
+      // validate all step fields and buttons
+      validateElements({
```

Comment:
what does performance look like here? esp for text fields where validation would now be triggered for each entry

Reply:
Good point.  We do need to revalidate all steps field and buttons on field value updates in order to get the behavior we want.  So, in order to make sure this is as fast as possible, I have implemented a 750ms debounce on the validateElements call in updateFieldValues.